infra/repository/github_environment_cd.tf

```diff
@@ -0,0 +1,18 @@
+resource "github_repository_environment" "github_repository_environment_prod_cd" {
+  environment = "prod-cd"
```

Comment:
what is the purpose of this environment?

Reply:
To manage azure login through federated credentials and deployment reviewers

Reply:
ok, but what are the differences between `prod-cd` and `prod-cd`?